.gitignore

```diff
@@ -5,3 +5,4 @@ __pycache__
 /yamllint.egg-info
 /build
 /.eggs
+.DS_Store
```

Comment:
I don't think this belongs in the repo-specific gitignore (because it shouldn't have to care about OS and editor specific junk files), but as a matter of practicality, I see no harm.

As always, whatever Adrien says.

Reply:
As you say, this `.DS_Store` has nothing to do with yamllint nor Python, so it should be excluded in a global system-specific conf.

Also, this is completely unrelated from the "float value linting" change.

Reply:
I see no harm in adding it, this is a pretty common annoyance; it is something I see in a lot of `.gitignore`s.

Reverted for now.